### DISCUSSION TITLE
Stream with bounding and motion boxes

### DISCUSSION THREAD

**Discussion by tmuellerchen** [blakeblackshear/frigate#2916] [Ask A Question]
Hi Frigate :)

is it possible to get the stream from debug mode with bounding an motion boxes?

I like to send it to a webpage-player

thanks in advance
thomas

**Comment by NickM-27** [comment]
At this point I don't believe so, [like it says here](https://docs.frigate.video/integrations/api#get-apicamera_name) in the docs the debug stream puts a lot of stress on the system and should just be used for debugging. Definitely not something to be streaming continuously.
